section2/try_ruby_lesson.rb

```diff
@@ -0,0 +1,146 @@
+# TryRuby (Practice Lesson) 
+# - Practice on JSON format Hash
+#   -- Display all book title
+#   -- Print & count books on specific year
+#   -- Show books in between years
+# - Using of Boolean Method
+# - Assigning a Blurb Class
+#   -- Adding Initialize
+#     --- limiting content length 
+#     --- initialize Time default
+#     --- add moodify method
+# - Storing all Blurb class
+
+
+# Practice on JSON format Hash
+def get_shakey
+  {
+    "William Shakespeare"=>
+    {
+    "1"=>  {"title"=>"The Two Gentlemen of Verona", "finished"=>1591}, 
+	  "2"=>  {"title"=>"The Taming of the Shrew", "finished"=>1591}, 
+	  "3"=>  {"title"=>"Henry VI, Part 2", "finished"=>1591}, 
+	  "4"=>  {"title"=>"Henry VI, Part 3", "finished"=>1591}, 
+	  "5"=>  {"title"=>"Henry VI, Part 1", "finished"=>1592}, 
+	  "6"=>  {"title"=>"Titus Andronicus", "finished"=>1592}, 
+	  "7"=>  {"title"=>"Richard III", "finished"=>1593}, 
+	  "8"=>  {"title"=>"Edward III", "finished"=>1593}, 
+	  "9"=>  {"title"=>"The Comedy of Errors", "finished"=>1594},
+	  "10"=> {"title"=>"Love's Labour's Lost", "finished"=>1595}
+    }
+  }
```

Comment:
@tlkiong do you mean for this hash, put it into one line? If I wanted the one line format like others?
eg: `.select { |key,value| value["finished"] == year }`

Reply:
no. i am asking you to have **consistency** in your code
![image](https://user-images.githubusercontent.com/5483026/128374006-a249c5df-a006-4993-90d7-0c69a459446b.png)